apps/devhub/templates/devhub/versions/list.html

```diff
@@ -277,6 +277,42 @@ <h3 data-tmpl="{{ _('Delete Version {version}') }}"></h3>
   </div>
   {% endif %}
 
+  {% if addon.is_listed %}
+  <div id="modal-unlist" class="modal">
+    <form method="post" action="{{ addon.get_dev_url('unlist') }}">
+      <h3>
+        {{ _('Unlist Add-on') }}
+      </h3>
+      <p>
+        {% trans %}
+          Unlisting your add-on will make it (and each of its versions/files)
+          invisible on this website. It won't show up in searches, won't have a
+          public facing page, and won't be updated automatically for current
+          users.  It is recommended for unlisted add-ons to provide a custom
+          <updateURL> (link to mdn) in their install.rdf file for automatic
+          updates.
+
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+          Are you sure you wish to unlist your add-on?
+        {% endtrans %}
+      </p>
+      {{ csrf() }}
+      <input type="hidden" name="version_id" class="version_id">
```

Comment:
I don't think we need the version (same for the `$('.version_id', this).val($(d.click_target).attr('data-version'));` in the modal declared below in the devhub.js file)